src/content/docs/extensions/json.mdx

```diff
@@ -537,7 +537,7 @@ Result:
 ```
 ┌───────┬────────────────────────────────────────────────┐
 │ p.id  │ p.description                                  │
-│ INT64 │ STRING                                         │
+│ INT64 │ json                                           │
```

Comment:
On current master branch, json data types now return json types instead of STRING